index.html

```diff
@@ -82,7 +82,17 @@
             note: "v1.0", w3cid: 102128},
           { name: "Kyle Den Hartog", url: "https://www.linkedin.com/in/kyledenhartog/",
             company: "MATTR", companyURL: "https://mattr.global/",
-            note: "v1.1", w3cid: 103517}
+            note: "v1.1", w3cid: 103517},
+          { name: "Orie Steele", url: "https://github.com/OR13",
+            company: "Transmute",
+            companyURL: "https://www.transmute.industries/",
+            note: "v2.0", w3cid: 109171},
+          { name: "Michael B. Jones", url: "https://self-issued.info/",
+            company: "Microsoft", companyURL: "https://www.microsoft.com",
+            note: "v2.0", w3cid: 38745},
+          { name: "Gabe Cohen", url: "https://github.com/decentralgabe",
+            company: "Block", companyURL: "https://block.xyz/",
+            note: "v2.0", w3cid: 0}
```

Comment:
@iherman, I need Gabe's `w3cid` value here.

Reply:
@msporny is it the ID here? `https://www.w3.org/users/116851`?

If so, `116851`.

Reply:
That's it! Thank you... only you and Ivan can see it. Adding it now.

Reply:
Done in 2ffb43dafd9cf7c179fd383e057f77e2eba03fa0.